docs/dev-guides/destroy_deployment.md

```diff
@@ -0,0 +1,227 @@
+[Return to Developer Guide](../dev.md)
+
+# **Destroy Deployments**
+
+It is essential to destroy a deployment at the end of its life cycle so that no resources are left in place. With a proper destruction procedure, one can mitigate the accrued cost of AWS. 
+
+---
+
+## **WARNING:** Before deleting the deployments, developers will need to account for the following: 
+
+- When deleting the `CloudFormation` stack, the deletion order matters.
+- Delete some of the `CloudFormation` stacks before deleting ECR.
+- The name of the items you're deleting may differ depending on the deployment you are taking down. For example, your deployment's `CloudFormation` stack may not have a `region` name or may not have a `<deployment-name>-auth` stack.
+- Do **NOT** take down the `Cognito` and `CloudWatch` logs. These are used for statistical analysis later on.
+
+---
+
+## **Removing CloudFormation Stacks**
+
+---
+
+When you first go to CloudFormation, it should look something like the following:
+
+![CloudForamtion Stacks](../assets/docs-cloudFormation-stacks.PNG)
+
+
+Because the order of removing deployment is essential, you will need to take down the `CloudFormation` stacks first. Deleting the following CloudFormation stacks will kill the deployment and remove its resources:
+
+- `<deployment_name>-container`
+- `<deployment_name>-auth`
+- `<deployment_name>-cluster`
+- `deployment_name`
+
+---
+
+_**NB**_: **Do not manually delete any S3 buckets after emptying them** in the steps below.
+
+If you delete the buckets after emptying them, the `CloudFormation` stack deletions associated with those buckets will fail. To fix the issues regarding deleted buckets, you will have to recreate the empty buckets to proceed. 
+
+Therefore, let `CloudFormation` delete the S3 buckets for you once you empty them.
+
+---
+
+Follow the below instructions to properly delete these stacks.
+
+---
+
+## **Steps to Delete `CloudFormation` Stacks**
+
+---
+
+### **1. Prepare to delete the `<deployment_name>-container` CloudFormation stack**
+    
+_NB: The `<deployment_name>-container` is independent of other stacks, i.e., the deletion order does not matter._
+
+1. Empty the `codepipeline-<region>-<deployment_name>-container` S3 bucket
+
+![empty container in s3](../assets/docs-s3-empty.PNG)
+
+2. Navigate to the AWS S3 console
+    1. Click the `codepipeline-<region>-<deployment_name>-container` S3 bucket option.
+    1. Click the `Empty` button
+        1. Confirm the deletion of bucket contents by typing `permanently delete` in the provided field
+        1. Click the `Empty` button
+
+### **2. Delete ECR repos for each profile**
```

Comment:
Technically, within JupyterHub a profile is a defined environment that the user chooses that includes an specific image. This image does not have to be unique to the profile. In fact, there are images in the repo that are not used in any profile like the `hub`, `cron`, etc. A more accurate wording for the header would simply be `Delete ECR repos`.